Problem3.java

```diff
@@ -0,0 +1,31 @@
+// Time Complexity : O(n)
+// Space Complexity : O(n)
```

Comment:
Space will be O(1) , in the worst case max characters in set can be valid ascii character i.e. 256 which is constant